docs/08-cicd.md

```diff
@@ -1,80 +1,190 @@
 # Continuous Integration and Delivery
 
-# Worker limits aka. $TOTAL_CPUS
+# Workflows
 
-# Pipeline Steps
+Several workflows exist for different purposes. See the table below for an overview.
 
-# Branches vs Pull Requests vs Scheduled
+|Workflow|Type-checking|Jest tests|Production build|E2E tests|Delivered in production|
```

Comment:
The term `Production build` suggests there is also such a thing as `Debug build`. Instead, the only difference is setting the environment variable `E2E_TESTING_IDS`.

Reply:
```suggestion
|Workflow|Type-checking|Jest tests|Production build|E2E tests|Delivered to production|
```